### PR TITLE
remove theme ID fetching from generation of Runner config

### PR DESCRIPTION
**Pull request by peetya**

### CODE REVIEW

File: models/classes/runner/config/QtiRunnerConfig.php [hunk 1]
```diff
@@ -38,7 +38,14 @@ class QtiRunnerConfig extends ConfigurableService implements RunnerConfig
 
     const OPTION_CONFIG = 'config';
 
+    /**
```
Comment (alexeyvy) [inline models/classes/runner/config/QtiRunnerConfig.php]: No usages within oat-sa repos, maybe we just remove it in this version?

https://github.com/search?q=org%3Aoat-sa+TOOL_ITEM_THEME_SWITCHER&type=Code

File: models/classes/runner/config/QtiRunnerConfig.php [hunk 2]
```diff
@@ -38,7 +38,14 @@ class QtiRunnerConfig extends ConfigurableService implements RunnerConfig
...
+     * @deprecated since version 29.2.0, to be removed in 30.0.0. Use QtiRunnerService::TOOL_ITEM_THEME_SWITCHER instead
+     */
     const TOOL_ITEM_THEME_SWITCHER = 'itemThemeSwitcher';
+
+    /**
```
Comment (alexeyvy) [inline models/classes/runner/config/QtiRunnerConfig.php]: Same thing: https://github.com/search?q=org%3Aoat-sa+TOOL_ITEM_THEME_SWITCHER_KEY&type=Code

File: models/classes/runner/config/QtiRunnerConfig.php (hunk 1 above)
Comment (peetya) [reply models/classes/runner/config/QtiRunnerConfig.php]: Community bundles could use it so it's more safe to mark it as deprecated.

File: models/classes/runner/config/QtiRunnerConfig.php (hunk 2 above)
Comment (peetya) [reply models/classes/runner/config/QtiRunnerConfig.php]: Same comment ;) So we cannot be sure that no other community extension is using that so it's more safe to mark it as deprecated.

**Comment by peetya** [comment]
I added some unit tests, which covers the updated part of the class.

**Pull request merged**